### PR TITLE
declare "reuse values" control mode in helmDefaults

### PR DESCRIPTION
**Pull request by kkoralsky**
adds option to declare `--reuse-values` control mode via `.helmDefaults.reuseValues` in `helmfile.yaml`

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -213,6 +213,8 @@ helmDefaults:
   # When set to `true`, skips running `helm dep up` and `helm dep build` on this release's chart.
   # Useful when the chart is broken, like seen in https://github.com/roboll/helmfile/issues/1547
   skipDeps: false
+  # if set to true, reuses the last release's values and merges them with ones provided in helmfile (default: false)
+  reuseValues: true
```
Comment (yxxhero) [inline docs/index.md:217]:
```suggestion
  # if set to true, reuses the last release's values and merges them with ones provided in helmfile (default: false)
  reuseValues: false
```

**Comment by yxxhero** [comment]
@kkoralsky please fix lint error.

**Comment by kkoralsky** [comment]
@yxxhero golangci-lint run should pass now

File: cmd/diff.go [hunk 2]
```diff
@@ -49,7 +49,7 @@ func NewDiffCmd(globalCfg *config.GlobalImpl) *cobra.Command {
 	f.StringVar(&diffOptions.Output, "output", "", "output format for diff plugin")
 	f.BoolVar(&diffOptions.SuppressSecrets, "suppress-secrets", false, "suppress secrets in the output. highly recommended to specify on CI/CD use-cases")
 	f.StringArrayVar(&diffOptions.Suppress, "suppress", nil, "suppress specified Kubernetes objects in the output. Can be provided multiple times. For example: --suppress KeycloakClient --suppress VaultSecret")
-	f.BoolVar(&diffOptions.ReuseValues, "reuse-values", false, "reuse the last release's values and merge in any overrides from other sources")
+	f.BoolVar(&diffOptions.ReuseValues, "reuse-values", false, `Override helmDefaults.reuseValues "helm upgrade --install --reuse-values"`)
```
Comment (yxxhero) [inline cmd/diff.go:52]:
```suggestion
	f.BoolVar(&diffOptions.ReuseValues, "reuse-values", false, `Override helmDefaults.reuseValues "helm diff upgrade --install --reuse-values"`)
```

**Review by yxxhero — APPROVED** [review]

**Comment by yxxhero** [comment]
@kkoralsky Thanks for your work.

**Pull request merged**